### ISSUE TITLE
[Feature request] Internationalization (i18n) & localisation (l10n) & translation (t9n)

### ISSUE THREAD

**Issue by cravend** [mTvare6/hello-world.rs#75]
When I first set out to study abroad, I knew it would be a life changing experience for me. Along the way I made new friends, tried new foods, and, most importantly: learned new languages.

As a result, I've decided to move on from my English-speaking past. I've successfully converted all of my personal stack to non-English except for one: this application.

My use case is: I use this script to fill my MOTD file (I call this as part of my pre-login RC file, I've found it really helps my oh-my-zsh loading times). Therefore, I get a personalised "Hello, world!" greeting when my terminal loads. However, I'd love to have this localized so that it greets me in non-English.

Surely there's some crate that lets us detect the computer's default language...

**Comment by cravend** [comment]
I've been informed by my eyes on the source code that this feature may exist, but you can't close this because I think my issue is better. Cheers!

**Comment by mTvare6** [comment]
yep, I won't close this because this is very important unlike other issues which makes fun of rust 😠  like #69

**Comment by DexterHaxxor** [comment]
W2n I f3t s1t o1t to s3y a4d, I k2w it w3d be a l2e c6g e8e f1r me. A3g t1e w1y I m2e n1w f5s, t3d n1w f3s, a1d, m2t i9y: l5d n1w l7s.

As a r4t, I've d5d to m2e on f2m my E5h-s6g p2t. I've s10y c7d a1l of my p6l s3k to n1n-E5h e4t f1r o1e: t2s a9n.

My u1e c2e is: I u1e t2s s4t to f2l my M2D f2e (I c2l t2s as p2t of my p1e-l3n RC f2e, I've f3d it r4y h3s my oh-my-z1h l5g t3s). T7e, I g1t a p10d "H3o, w3d!" g6g w2n my t6l l3s. H5r, I'd l2e to h2e t2s l7d so t2t it g4s me in n1n-E5h.

S4y t3e's s2e c3e t2t l2s us d4t t1e c6r's d5t l6e...
